tests/conftest.py

```diff
@@ -0,0 +1,26 @@
+import pytest
+from pathlib import Path
+import json
+from jsonschema import validate
+from jsonschema.validators import (
+    Draft7Validator,
+    validator_for,
+)
+
+
+@pytest.fixture
+def schema():
+    p = Path('.') 
```

Comment:
Why not just instantiate the whole path here?

Also, you probably shouldn't make assumptions about what the current directory is.  The more robust way to do this is anchoring to `__file__`


```python
ROOT = Path(__file__).parent
path = ROOT / <whatever>
```

This still depends on the location of the actual python module but it is more reliable.